test/lint/lint-python.sh

```diff
@@ -31,7 +31,6 @@ export LC_ALL=C
 # E401 multiple imports on one line
 # E402 module level import not at top of file
 # F403 'from foo_module import *' used; unable to detect undefined names
-# F405 foo_function may be undefined, or defined from star imports: bar_module
```

Comment:
If you disable the rule just where it's incoherent, then you can leave this active for other uses.